rdt-app/app/src/main/java/io/ona/rdt_app/activity/MainActivity.java

```diff
@@ -10,44 +11,49 @@
 import android.util.Log;
 import android.view.View;
 
-import com.crashlytics.android.Crashlytics;
-
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import io.fabric.sdk.android.Fabric;
 import io.ona.rdt_app.R;
+import io.ona.rdt_app.util.Constants;
 import util.RDTCaptureJsonFormUtils;
 
-import static io.ona.rdt_app.activity.Constants.REQUEST_CODE_GET_JSON;
+import static io.ona.rdt_app.util.Constants.REQUEST_CODE_GET_JSON;
 
 public class MainActivity extends AppCompatActivity {
 
     private RDTCaptureJsonFormUtils jsonFormUtils;
     private final String TAG = MainActivity.class.getName();
+    private Activity context;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        Fabric.with(this, new Crashlytics());
```

Comment:
Why remove remove, should be remain but enabled only for Release builds which are shared with bigger team for testing